lib/tasks/partners.rake

```diff
@@ -0,0 +1,23 @@
+namespace :partners do
+  desc 'Provision dummy IAL2 users from CSV file'
```

Comment:
One thing that just occurred to me is that this doesn't _necessarily_ need to be for IAL2 users, it would be easy to just pass an email domain and a count and have it do the same thing just for IAL1. I don't think that's necessary for this PR but just a thought around the naming of the task.